contributing.md

```diff
@@ -16,7 +16,8 @@ stylistic, refactoring, or "cleanup" changes).
 Please check with us on the
 [dev list](https://groups.google.com/forum/#!forum/bazel-dev) before investing
 a lot of time in a patch. Meet other Bazel contributors on [IRC](http://webchat.freenode.net)
-(irc.freenode.net#bazel).
+(irc.freenode.net#bazel) or [Slack](https://bazelbuild.slack.com) (get an
```

Comment:
Can we list slack first and IRC second.
And.... we should us the other invite mechanism rather than herokuapp.


https://join.slack.com/t/bazelbuild/shared_invite/enQtNDYyODAxNDY4MjQxLWU1OTBjOWJjOTA1MmE5YjIwMTM5MjFhZGMwM2YzYTEzYTdhNGIzMTFiZjI1NTlhYTkzNGQ5MjBkYzE5YjMyMTE

And we also have to write our own invite server.

Reply:
Acutally, I think we should drop IRC from contributing. Bazel developers should have one authoritative place to go.

Reply:
done.